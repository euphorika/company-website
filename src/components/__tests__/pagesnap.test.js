import React from "react"
import { mount } from "enzyme"
import toJson from 'enzyme-to-json'

import PageSnapContainer from "../pagesnap"

describe("PageSnapContainer", () => {
  it("renders correctly", () => {
    const wrapper = mount(<PageSnapContainer><div>Snap Entry 1</div><div>Snap Entry 2</div></PageSnapContainer>)
    expect(wrapper).toMatchSnapshot()
  })
})
