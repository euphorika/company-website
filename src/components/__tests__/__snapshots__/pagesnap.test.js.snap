// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageSnapContainer renders correctly 1`] = `
<PageSnapContainer>
  <div
    className="pageSnapContainer"
  >
    <div>
      Snap Entry 1
    </div>
    <div>
      Snap Entry 2
    </div>
  </div>
</PageSnapContainer>
`;
